Sources/Internal/FrameProvider.swift

```diff
@@ -77,6 +79,16 @@ final class FrameProvider {
       let x = minXOfMonth(containingItemWithFrame: layoutItem.frame, at: position)
       let y = minYOfMonth(containingDayItemWithFrame: layoutItem.frame, atRowInMonth: rowInMonth)
       return CGPoint(x: x, y: y)
+        
+    case .monthFooter(let month):
+      let x = layoutItem.frame.origin.x
+        let y = layoutItem.frame.minY -
```

Comment:
The indentation is a bit off here. I need to update the style guidelines so that contributors use 2-space indentation instead of the default 4. Personally, I like 4, but all of Airbnb's iOS stuff is 2, so I'd like to stick with that for now.

Reply:
I personally also like 4, but can fix the indention now!